diff --git a/node_modules/speechmarkdown-js/dist/src/SpeechMarkdownGrammar.js b/node_modules/speechmarkdown-js/dist/src/SpeechMarkdownGrammar.js
index 59ebb35..bc86ec8 100644
--- a/node_modules/speechmarkdown-js/dist/src/SpeechMarkdownGrammar.js
+++ b/node_modules/speechmarkdown-js/dist/src/SpeechMarkdownGrammar.js
@@ -67,7 +67,7 @@ function speechMarkdownGrammar(myna) {
         // Emphasis
         // The emphasis tag should be preluded and followed by a not-letter-character.
         // Otherwise an example like above would be captured.
-        var notLetterChar = m.not(m.letters);
+        var notLetterChar = m.not(m.choice(m.letters, m.digits));
         this.shortEmphasisModerate = m.seq(notLetterChar, '+', this.plainTextEmphasis, '+', notLetterChar).ast;
         this.shortEmphasisStrong = m.seq(notLetterChar, '++', this.plainTextEmphasis, '++', notLetterChar).ast;
         this.shortEmphasisNone = m.seq(notLetterChar, '~', this.plainTextEmphasis, '~', notLetterChar).ast;
